projects/ngx-mask-lib/src/lib/ngx-mask-applier.service.ts

```diff
@@ -724,6 +724,23 @@ export class NgxMaskApplierService {
         if (result.length === 0) {
             res = !this.dropSpecialCharacters ? `${this.prefix}${result}` : `${result}`;
         }
+
+        const indexFirstNonSpecialCharacter = Array.from(this.maskExpression).findIndex(
```

Comment:
could be something like this 
let indexFirstNonSpecialCharacter = -1;

for (let i = 0; i < maskExpression.length; i++) {
    if (!specialCharacters.includes(maskExpression[i])) {
        indexFirstNonSpecialCharacter = i;
        break;
    }
}